AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/app.txt

```diff
@@ -4,193 +4,258 @@
+                            "param": "kernel",
+                            "value": "linear"
                         }
                     },
-                    "selected": false
+                    "inputs": [
+                        {
+                            "name": "training_data",
+                            "id": "training_data",
+                            "type": "DataFrame|Matrix",
+                            "multiple": false
+                        },
+                        {
+                            "name": "input_data",
+                            "id": "input_data",
+                            "type": "DataFrame|Matrix",
+                            "multiple": false
+                        }
+                    ],
+                    "outputs": [
+                        {
+                            "name": "default",
+                            "id": "default",
+                            "type": "DataFrame"
+                        }
+                    ],
+                    "path": "PYTHON/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.py",
+                    "selected": true
                 },
                 "position": {
-                    "x": 30.299117370698312,
-                    "y": 176.16569145577284
+                    "x": 928.4663087537485,
+                    "y": 682.0331845444681
                 },
-                "selected": false,
+                "selected": true,
                 "positionAbsolute": {
-                    "x": 30.299117370698312,
-                    "y": 176.16569145577284
+                    "x": 928.4663087537485,
+                    "y": 682.0331845444681
                 },
                 "dragging": true
             },
             {
                 "width": 150,
                 "height": 150,
-                "id": "READ_CSV-46e0dbba-c159-4856-b711-a391ff19d5ea",
+                "id": "READ_CSV-59c660d6-c37c-48e6-9e32-11490ebba845",
                 "type": "FILE",
                 "data": {
-                    "id": "READ_CSV-46e0dbba-c159-4856-b711-a391ff19d5ea",
-                    "label": "READ CSV 1",
+                    "id": "READ_CSV-59c660d6-c37c-48e6-9e32-11490ebba845",
+                    "label": "READ CSV",
                     "func": "READ_CSV",
                     "type": "FILE",
                     "ctrls": {
                         "file_path": {
+                            "type": "str",
+                            "default": "https://raw.githubusercontent.com/cs109/2014_data/master/countries.csv",
                             "functionName": "READ_CSV",
                             "param": "file_path",
-                            "value": "~/Downloads/iris_test_no_label.csv"
+                            "value": "/Users/nick/Desktop/flojoy_dev/studio/PYTHON/nodes/EXTRACTORS/FILE/READ_CSV/assets/iris_train.csv"
```

Comment:
Please use a relative path so that it works on any machine. e.x `PYTHON/nodes/EXTRACTORS/FILE/READ_CSV/assets/iris_train.csv`